### ISSUE TITLE
When exporting jar from explorer title, the export jar task is available

### ISSUE THREAD

**Issue by CsCherrYY** [microsoft/vscode-java-dependency#495]
Currently the expect behavior of exporting jar is singleton and multiple exporting jar tasks are not supported at the same time. But when the exporting jar process comes from explorer's title, the export jar task is available.

**Comment by jdneo** [comment]
> multiple exporting jar tasks are not supported at the same time

What will happen if multiple exporting jar tasks are undergoing? Sounds like that could be a feature actually. 😄

**Comment by CsCherrYY** [comment]
Yes. Actually we should create a terminal and "delegate" the export task to it. A new export task should not influence the existing tasks and should be executed separately.

**Comment by jdneo** [comment]
Different tasks go to different terminals, is that what you mean?
